core-bundle/src/Migration/AbstractMigration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration;
+
+abstract class AbstractMigration implements MigrationInterface
+{
+    public function getName(): string
+    {
+        return str_replace('\\', ' ', static::class);
+    }
+
+    public function createResult(bool $successful = true, string $message = null): MigrationResult
```

Comment:
shouldn't this method be `protected`? I also find it confusing that I can create a result without giving any arguments. Maybe `$successful` shouldn't have a value…

Reply:
Agree, will fix.

Reply:
Done in 47854333bba2587d8477bede3c8c3610e9a1b237